mava/configs/system/q_learning/rec_iql.yaml

```
@@ -0,0 +1,33 @@
+# --- Defaults REC-IDQN ---
+total_timesteps: 5_000_000
+num_updates: 1000 # total_timesteps takes preference
+seed: 1
+
+# --- Agent observations ---
+add_agent_id: True
+
+# --- RL hyperparameters ---
+buffer_min_size: 32
+update_batch_size: 1 # Number of vectorised gradient updates per device.
+
+rollout_length: 2 # Number of environment steps per vectorised environment.
+epochs: 2 # Number of learn epochs per training data batch.
```

Can we set these to good defaults, I found that with SAC 32-64 is a good value, the higher to more sample efficient we should be
```suggestion
epochs: 32 # Number of learn epochs per training data batch.
```

---

IQL has been doing well even without this change, but it's a good catch! Ideally, I'd actually want to base epochs off the amount of experience per step, because having that ratio be explicit is good for diagnosing why the system trains the way it does. Shall I just set the defaults differently for now?